app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_minicart.less

```diff
@@ -431,6 +431,33 @@
     }
 }
 
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
+    .minicart-items .item-qty {
+        margin: 0 @indent__s @indent__xs 0;
+    }
+}
+
+@media only screen 
+and (min-device-width: @screen__xs) 
+and (max-device-width: @screen__m) {
```

Comment:
This has to use two separate `.media-width`, one for minimum `@screen__xs` and one for minimum `@screen__m` that sets those values back.

Reply:
Maybe what I wrote is not clear enough here. I would like you to split this media query into 2 separate ones so we can use `.media-width` is such a way:
```less
.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__xs) {
  // your styles with margin and padding.
}

.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
  // styles reverting margin and padding for bigger screens.
}
```

Reply:
This one is still not addressed.